### DISCUSSION TITLE
NanoPi R6s什么时候可以支持哇？

### DISCUSSION THREAD

**Discussion by realwanggj** [coolsnowwolf/lede#10414] [Q&A / 快速问答]
没忍住搞了，主要也是为了升级2.5G和以后用HomeAssitants

**Comment by Alexrobot03** [comment]
同问，昨天刚入，老哥有没有组织

**Reply by realwanggj** [reply]
还没找到，[kiddin9](https://github.com/kiddin9)有。

**Comment by 0x676e67** [comment]
估计没有了，人太少了，厂家又不舍得花钱给开发者

**Reply by realwanggj** [reply]
希望能有吧。

**Comment by Aes64X** [comment]
你们几个可真秀啊，这价格，这性能，这外围接口。。。。一个x86拼装都出来了，还没成堆的补丁，套上内核就能用。

**Reply by Alexrobot03** [reply]
天天玩电脑X86都玩腻了，软路由再来个X86，兴致确实不高啊~再说了arm小巧，外观精致，奈斯~

**Comment by Aes64X** [comment]
退一步讲。。。上面几位就是买ITX或者大板规格的3588也是能理解的，买R6S.........看不懂。

**Reply by 0x676e67** [reply]
> 退一步讲。。。上面几位就是买ITX或者大板规格的3588也是能理解的，买R6S…………看不懂。

有钱人的世界不懂，可以理解

**Reply by Alexrobot03** [reply]
大板规格都还是开发板啊，自己套外壳么~大板不就多了几个接口嘛

**Reply by Aes64X** [reply]
> 大板规格都还是开发板啊，自己套外壳么~大板不就多了几个接口嘛

接口多才有得玩，你看看R6S都啥外围，USB，EMMC，TF，没了。。。。。就那点电压，东西挂多了还带不动，玩个锤子。
大板的搞搞还能上个华为的SD100 6口万兆，那玩意x86用不了，只能ARM用。

**Reply by Alexrobot03** [reply]
软路由就好好当软路由～扩展对我来说没用

**Comment by 0x676e67** [comment]
笑死，你也不看看RK3588大板子多少钱，等会又说有这个钱早就去整x86了。每个人的需求都不一样，胡萝卜青菜各有所爱，没有什么好争的，别人有钱就是爱玩，非要扯什么接口，性价比。这跟玩arm（都玩arm了还考虑什么性价比？），总会跳出一群人 “不如x86” 有什么区别，人嘛有钱爱玩啥开心就行，扯太多显得自己高水平也没意义。

**Reply by Cathgao** [reply]
有钱爱玩怎么不花钱雇人做固件？有钱还要白嫖？

**Reply by 0x676e67** [reply]
> 有钱爱玩怎么不花钱雇人做固件？有钱还要白嫖？

你又来插什么嘴，碍你事了？你捐赠了吗？你怕是连50都不愿意进大雕 VIP群吧？

**Reply by 0x676e67** [reply]
> 有钱爱玩怎么不花钱雇人做固件？有钱还要白嫖？

如果没有，请你闭嘴

**Reply by Cathgao** [reply]
> > 有钱爱玩怎么不花钱雇人做固件？有钱还要白嫖？
> 
> 你又来插什么嘴，碍你事了？你捐赠了吗？你怕是连50都不愿意进大雕 VIP群吧？

暴露了到底是谁没捐赠还要装有钱人，笑死

**Reply by Cathgao** [reply]
一边又说人家就是有钱爱买来玩，一边又没钱捐赠来出固件天天搁着叫，没钱装什么吹什么逼，还装一个理中客在这自己水平最高的那个。
我就是你认为的捐赠爹，我就不闭嘴😄

**Reply by 0x676e67** [reply]
> 一边又说人家就是有钱爱买来玩，一边又没钱捐赠来出固件天天搁着叫，没钱装什么吹什么逼，还装一个理中客在这自己水平最高的那个。
> 我就是你认为的捐赠爹，我就不闭嘴😄

别人买什么关你屁事，只会指点江山叫叫叫是吧

**Reply by 0x676e67** [reply]
你看看这里评论有几个在指点江山，爱买什么还需要你管？我就是买了R6S怎么了？我有没有不尊重开发者？省得你在这叫？

**Reply by 0x676e67** [reply]
> 一边又说人家就是有钱爱买来玩，一边又没钱捐赠来出固件天天搁着叫，没钱装什么吹什么逼，还装一个理中客在这自己水平最高的那个。
> 我就是你认为的捐赠爹，我就不闭嘴😄

没买就别搁着叫了，恶心，一幅欠你钱一样。L大都没有说什么。

**Reply by 0x676e67** [reply]
> 一边又说人家就是有钱爱买来玩，一边又没钱捐赠来出固件天天搁着叫，没钱装什么吹什么逼，还装一个理中客在这自己水平最高的那个。
> 我就是你认为的捐赠爹，我就不闭嘴😄

再说，L大开VIP群我马上进，你又不是开发者，在这叫什么。

**Reply by 0x676e67** [reply]
> 一边又说人家就是有钱爱买来玩，一边又没钱捐赠来出固件天天搁着叫，没钱装什么吹什么逼，还装一个理中客在这自己水平最高的那个。
> 我就是你认为的捐赠爹，我就不闭嘴😄
还有你那只眼看见，要出固件在这叫了。屎可以乱吃，话不能乱讲

**Reply by 0x676e67** [reply]
> 笑死，你也不看看RK3588大板子多少钱，等会又说有这个钱早就去整x86了。每个人的需求都不一样，胡萝卜青菜各有所爱，没有什么好争的，别人有钱就是爱玩，非要扯什么接口，性价比。这跟玩arm（都玩arm了还考虑什么性价比？），总会跳出一群人 “不如x86” 有什么区别，人嘛有钱爱玩啥开心就行，扯太多显得自己高水平也没意义。

大家都是抱着期待看L大能不能做固件而已，说得难听点不给钱大家都是白嫖，前提是尊重开发者。看你好像是咬定了我来嚷嚷喊大雕开发，请你看清楚上下文，我回的是什么。上面那个指点江山，喊什么买x86（有性价比），买r6s（没性价比）就是脑子有问题的意思，。你看看你回的是什么玩意

**Reply by 1715173329** [reply]
好牛逼啊

**Reply by 0x676e67** [reply]
> 好牛逼啊

天灵佬半夜还回消息🙃

**Reply by realwanggj** [reply]
仅仅就是问问支持计划，给钱我也觉得很正常合理，我也给了，自己编译我也看了，我也只是期盼有L大支持优化。“人傻钱多”“白嫖”的帽子一个都没少扣，至于吗？真的伤害到那些给别人扣帽子的人了吗？难听的话不说了，不想给真正的开发者添堵，也不想成为在网上随便扣帽子张嘴就来的人。

**Comment by ForgotFun** [comment]
佐大原本是想过来劝架的，但看了一圈一想还是算了。大家还是参加佐大的OpenWrt培训班吧，这才是解决矛盾的最佳方案。

**Reply by realwanggj** [reply]
我本意也就是问问大佐我的这个问题有没有解决办法和找一些和我有同样问题的人。对于别的言论，我肯定是懒得看的，更懒得去产生矛盾。不过这种直接质疑别人立场或者问题的言论，哪都有。大佐没必要劝，这种争论本身就没有任何意义。我也仅仅是希望大佐如果可以的话，告诉我解决方案而已。

**Reply by realwanggj** [reply]
但是如果言论本身没给别人挂上别的帽子（“例如人傻钱多”），说说也没啥，但是一般给不出建设意见的人往往都会攻击对方立场。

**Comment by moumou2014** [comment]
跪求lean大神适配一下r6s的固件，大神能不能给个充电的链接，让我们给大神充充电。

**Reply by moumou2014** [reply]
哈哈哈，刚刚才看到有充电链接，给大神充一下

**Reply by realwanggj** [reply]
我也冲了

**Reply by moumou2014** [reply]
太好了，等冲的人多了，L大神一高兴就适配了呢，我玩这个纯粹是因为它比我那4125铁坨坨迷你太多了。
